src/plugins/profiling/server/routes/search_flamechart.ts

```diff
@@ -288,3 +294,11 @@ export function registerFlameChartSearchRoute(router: IRouter<DataRequestHandler
     }
   );
 }
+
+const now = () => {
```

Comment:
These were exported for a reason :-) Need them at several other places later. We can change that later.

Reply:
In the second commit I deleted those functions and didn't find any error... they apparently were used only inside the `search_flamechart` file?